### PR TITLE
ruben/eng-312/blocking create npm run scanaddress command v2

### PR DESCRIPTION
**Pull request by topether21**
Example:

`npm run hunter:bulk-transfer bc1pfvh396m58axyknye74zjuyt496y78xl9xccpzyhlzf8k93xx3ezqxel8zf bc1ptjcfjm8f7gz8umrh0sg3ve8ewq3gm0q9m7khhplny8u7m0nvqf8s0ym0zh vintage 5 90`

IMG_C821148B1634-1

IMG_55DE7B08E5B7-1

### CODE REVIEW

**Review by habibitcoin — APPROVED** [review]
Working fantastic!!

File: commands.js [hunk 1]
```diff
@@ -68,7 +69,96 @@ const create_withdraw_request = async (name, amount) => {
...
+                console.log(`Error checking bulk transfer with id: ${scan_request_id}: ${err.message}`);
+                clearInterval(intervalId); // Ensure the interval is cleared on error
+                reject(err); // Reject the promise
+            }
+        }, 1000); // Check every second
+    });
+}
+
+const bulk_transfer = async (p_from_address, p_to_address, p_tag_to_extract, p_num_of_tag_to_send, p_fee_rate) => {
+    try {
```
Comment (dannydeezy) [inline commands.js:108]: @habibitcoin @topether21 
Can we send a telegram notification here that just says: "Scan has been initiated, see terminal logs to track progress..." or something

(I'm running a scan on our many thousand utxos and its taking a while)

**Comment by dannydeezy** [comment]
mostly looks good - we should fast follow with year-based uncommons 
often people want to transfer something like "100 uncommons from year 2014 or higher"

**Review by dannydeezy — APPROVED** [review]

File: commands.js (hunk 1 above)
Comment (habibitcoin) [reply commands.js:108]: Yup, we'll add as a follow up

**Pull request merged**